packages/components/src/progress-bar/README.md

```diff
@@ -1,13 +1,51 @@
 # ProgressBar
 
-<div class="callout callout-alert">
-This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
-</div>
```

Comment:
I think the stabilization needs its own CHANGELOG entry.

Reply:
I thought it was an internal thing and shouldn't be part of the changelog.